scripts/india_rbi/below_poverty_line/BelowPovertyLine_India.tmcf

```diff
@@ -0,0 +1,71 @@
+Node: E:BelowPowertyLine_India->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_Rural_BelowPovertyLevelInThePast12Months
+value: C:BelowPowertyLine_India->count_person_rural
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+Node: E:BelowPowertyLine_India->E2
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_Urban_BelowPovertyLevelInThePast12Months
+value: C:BelowPowertyLine_India->count_person_urban
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E3
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_BelowPovertyLevelInThePast12Months
+value: C:BelowPowertyLine_India->count_person_combined
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E4
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_Rural_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person_Rural
+value: C:BelowPowertyLine_India->percentage_person_rural
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E5
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_Rural_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person_Urban
+value: C:BelowPowertyLine_India->percentage_person_urban
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E6
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_Person_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person
+value: C:BelowPowertyLine_India->percentage_person_combined
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E7
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:PovertyLine_Person_Rural
+value: C:BelowPowertyLine_India->poverty_line_rural
+unit: dcs:IndianRupee
+observationDate: C:BelowPowertyLine_India->year
+observationPeriod: "P1Y"
+observationAbout: C:BelowPowertyLine_India->territory
+
+
+Node: E:BelowPowertyLine_India->E8
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:PovertyLine_Person_Urban
```

Comment:
If you see my other comment that it seems "Urban Poverty Line of a place" is a real concept, then maybe here we could say `variableMeasured: dcs:urbanPovertyLine` and likewise for Rural/Total

Reply:
Done. As per our discussion.